test/Controller/ArticleControllerTest.php

```diff
@@ -564,6 +564,8 @@ public function it_has_schema_org_metadata()
 
         $this->assertEquals('http://schema.org/ScholarlyArticle', $node->getType()->getId());
         $this->assertEquals(new TypedValue('Article title', RdfConstants::XSD_STRING), $node->getProperty('http://schema.org/headline'));
+
+        $this->markTestIncomplete('This test fails if schema.org is broken!');
```

Comment:
This gives us the opportunity to refactor the test to not need a http request in future but won't block current progress on other PR's.